### ISSUE TITLE
So is ASP.NET on .NET Framework being discontinued?

### ISSUE THREAD

**Issue by MikeChristensen** [aspnet/AspNetCore.Docs#15411]
The documentation states ASP.NET Core 3.0 will only work on .NET Core, rather than .NET Standard.  Does this mean ASP.NET is being discontinued on the full .NET Platform?  Or, will ASP.NET (Non-Core) be coming back?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ad8c13e7-78e6-cf9a-e7cb-3215df5fd1c4
* Version Independent ID: a3c1d2a6-625a-3409-120d-8372230a8196
* Content: [Introduction to ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by scottaddie** [comment]
The documentation to which you're referring applies to ASP.NET Core, not ASP.NET 4.x. You can find the official ASP.NET 4.x support policy info and retirement dates at https://dotnet.microsoft.com/platform/support/policy/aspnet. There are no plans to breathe new life into ASP.NET 4.x. The team's engineering efforts are focused on cross-platform ASP.NET (ASP.NET Core).

As of version 3.0 of ASP.NET Core, an ASP.NET Core project can only target .NET Core. The reasoning is that ASP.NET Core 3.0 adopts .NET Standard 2.1. .NET Framework doesn't implement .NET Standard 2.1; .NET Core 3.0 does implement .NET Standard 2.1.

**Comment by scottaddie** [comment]
@MikeChristensen Yes, .NET 5.0 will be that unified, cross-platform .NET future you're describing. .NET Framework will continue to ship as a critical component of Windows and will therefore continue to be supported.

For any greenfield development, .NET Core is the direction customers are encouraged to go. Common reasons for adopting .NET Core include:

- cross-platform support
- **significant** performance improvements
- support for installing multiple versions of .NET Core side-by-side on the same machine without creating conflicts
